Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst

```diff
@@ -1,2 +1,2 @@
-Fix an assertion failure in case of an exception with a bad ``__module__``
-attribute. Patch by Oren Milman.
+Fix an assertion failure in `c.PyErr_WriteUnraisable()` in case of an
```

Comment:
What is `c.PyErr_WriteUnraisable()`?

Reply:
https://devguide.python.org/committing/#what-s-new-and-news-entries says:
`(single backticks) in reST can be used to refer to objects in the documentation.`
I thought this means that i should specify (inside single backticks) an object in the docs, according to how it appears in the docs (i.e. when pressing the `Permalink to this definition` button - https://docs.python.org/3.7/c-api/exceptions.html#c.PyErr_WriteUnraisable).

so it should just be `PyErr_WriteUnraisable()`?

Reply:
Yes, it should be without "c." prefix.